engine/runtime/src/test/java/org/enso/compiler/SerdeCompilerTest.java

```diff
@@ -115,6 +116,7 @@ private Context ensoContextForPackage(String name, File pkgFile, boolean disable
             .option(
                 RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
                 Paths.get("../../distribution/component").toFile().getAbsolutePath())
+            .option(RuntimeOptions.LOG_LEVEL, Level.WARNING.getName())
```

Comment:
I don't understand what is the idea of the fix? I don't see any change in the code, just in the tests. Thus the only idea I see is: _Increase logging level, so logging doesn't happen_. But I wanted the logging to happen when I faced the issue.

Reply:
Yes, this was only a _partial_ fix for some of the tests. It wasn't addressing the core of the problem. Now it does.